spec/timezone.html

```diff
@@ -134,7 +134,18 @@ <h1>Temporal.TimeZone.prototype.getOffsetNanosecondsFor ( _absolute_ )</h1>
         1. Let _timeZone_ be the *this* value.
         1. Perform ? RequireInternalSlot(_timeZone_, [[InitializedTemporalTimeZone]]).
         1. Perform ? RequireInternalSlot(_absolute_, [[InitializedTemporalAbsolute]]).
-        1. Return ? GetTimeZoneOffsetNanoseconds(_absolute_.[[Nanoseconds]], _timeZone_.[[Identifier]]).
+        1. Let _offset_ be ! ParseOffsetNanoseconds(_timeZone_.[[Identifier]]).
+        1. If _offset_ is not *undefined*, then
+          1. Return _offset_.
+        1. Let _epochNs_ be _absolute_.[[Nanoseconds]].
+        <!-- Can this be BalanceDateTime(1970, 1, 1, 0, 0, 0, 0, 0, _epochNs_)? -->
+        1. Let _ms_ be RoundTowardsZero(_epochNs_ / 1,000,000<sub>ℝ</sub>).
+        1. Let _mcs_ be ! NonNegativeModulo(RoundTowardsZero(_epochNs_ / 1000<sub>ℝ</sub>), 1000<sub>ℝ</sub>).
+        1. Let _ns_ be ! NonNegativeModulo(_epochNs_, 1000<sub>ℝ</sub>).
+        1. Let _tm_ be ! ToLocalTime(_ms_, *"gregory"*, _timeZone_.[[Identifier]]).
+        1. Let _result_ be ? BalanceDateTime(_tm_.[[Year]], _tm_.[[Month]], _tm_.[[Day]], _tm_.[[Hour]], _tm_.[[Minute]], _tm_.[[Second]], _ms_, _mcs_, _ns_).
```

Comment:
I'm late to the review party on this PR, but I didn't understand this line. Won't the `BalanceDateTime` operation end up almost doubling the distance from epoch in milliseconds?  Is there a missing step where we need to calculate the modulo `_ms_` use that remainder in `BalanceDateTime` instead of `_ms_`?

Reply:
Good catch, thank you: https://github.com/tc39/proposal-temporal/pull/828